pkg/virt-controller/services/template_test.go

```diff
@@ -1964,69 +1945,45 @@ var _ = Describe("Template", func() {
 				}
 			})
 			It("should add 1G of memory overhead", func() {
-				sriovInterface := v1.InterfaceSRIOV{}
-				domain := v1.DomainSpec{
-					Resources: v1.ResourceRequirements{
-						Requests: kubev1.ResourceList{
-							kubev1.ResourceMemory: resource.MustParse("1G"),
-						},
-					},
-				}
-				domain.Devices.Interfaces = []v1.Interface{{Name: "testnet", InterfaceBindingMethod: v1.InterfaceBindingMethod{SRIOV: &sriovInterface}}}
-				vmi := v1.VirtualMachineInstance{
-					ObjectMeta: metav1.ObjectMeta{
-						Name: "testvmi", Namespace: "default", UID: "1234",
+				vmi := newVMIWithSriovInterface("testvmi", "1234")
+				vmi.Spec.Domain.Resources = v1.ResourceRequirements{
+					Requests: kubev1.ResourceList{
+						kubev1.ResourceMemory: resource.MustParse("1G"),
 					},
-					Spec: v1.VirtualMachineInstanceSpec{Domain: domain},
 				}
+				domain := vmi.Spec.Domain
```

Comment:
Why do you need this? It seems to me like it can be inlined.

Reply:
Done